src/anim/binder/default-anim-binder.js

```diff
@@ -124,28 +128,48 @@ class DefaultAnimBinder {
         };
     }
 
-    resolve(path) {
-        var propertyLocation = AnimBinder.decode(path);
-
-        var node = this.graph.findByPath(propertyLocation.entityPath[0]);
+    findNode(graph, path) {
+        var node = graph.findByPath(path.entityPath.join('/'));
```

Comment:
it might be worth adding a `findByPath` function to graph-node which takes the array and uses it directly instead of joining into a string here (which won't handle escapes properly) just to split in the function. @willeastcott would you be ok adding a function to `GraphNode` or so we keep such a function here in `default-anim-binder`?

Reply:
I've updated the graph.findByPath function to support arrays too.